spec/models/vehicle_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Vehicle, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
I suggest removing this file altogether if it's not used.
I don't think models tests are always necessary if the model has no logic in it. And the validation is tested by other tests